Json.More/Json.More.csproj

```diff
@@ -34,7 +34,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
-    <PackageReference Include="System.Text.Json" Version="6.0.0" />
+    <PackageReference Include="System.Text.Json" Version="8.0.0" />
   </ItemGroup>
```

Comment:
Latest is 8.0.1, can we just specify it without any Condition so we know we're on the same version across all flavors?

Reply:
Well... is it really necessary to specify it at all in .Net 8?  If you're not in .Net 8, you get the .Net Standard 2.0 lib.  But if you're _in_ .Net 8, then the package is built in, and the reference isn't necessary.

Reply:
But in NET8 which version are we getting? Is it always latest? I suspect it's tied to whatever version of the SDK we compiled against. Having one be explicitly specified and the other implicit seems like a recipe for getting inconsistent versions between net6 and net8 and getting bugs that'll make you tear your hair out.

Reply:
(I'm just addressing the build condition.  I'm happy with updating the reference to current latest.)

There's nothing guaranteeing that this library will remain updated with the latest either.  If you bump your local framework to 8.0.3, but this lib hasn't updated yet, it'll still use the 8.0.1 from the package.  I'd prefer it use the local framework.  If the local framework is before 8.0.0, then they'll get the .Net Standard version of this lib along with whatever stj8 version this lib needs.